travis/tc3_style/check.sh

```diff
@@ -4,28 +4,40 @@ source $CI_HELPER_PATH/travis/settings.sh
 
 exit_code=0
 
-_header "Checking that source code files do not contain tabs..."
+_header "Checking that source code files do not contain leading tabs..."
 
-tab_lines=$(./files.sh | xargs awk '/\t/' | wc -l)
-if [ "${tab_lines}" -ne 0 ]; then
-  echo "Found ${tab_lines} lines with tabs"
+tab_lines=$(./files.sh | xargs egrep $'^\s*\t')
```

Comment:
Much improved regex 👍

Reply:
Is `egrep -n` portable for showing line numbers?

Reply:
I don't see why it wouldn't be. I'll put that in there, it will make the error output more readable.

Reply:
I'll also take this opportunity to switch out `egrep` for `grep -E` just in case there are portability concerns.